packages/manager/apps/ips/src/routes/routes.constant.ts

```diff
@@ -1,14 +1,17 @@
 export const subRoutes = {
-  root: '/ip',
+  root: '',
+  listing: '/ip',
```

Comment:
```suggestion
  root: '/ip',
```

Need to keep this or this will break container url redirection in staging/prod